packages/next-on-pages/src/buildApplication/processVercelFunctions/dedupeEdgeFunctions.ts

```diff
@@ -201,6 +210,9 @@ async function buildFunctionFile(
 	const finalFileContents = `${functionImports}${fileContents}`;
 	const buildPromise = buildFile(finalFileContents, newFnPath, {
 		relativeTo: nopDistDir,
+	}).then(async () => {
+		const { size } = await stat(newFnPath);
+		fnInfo.outputSize = size;
```

Comment:
I think output size is very generic, could you do `outputByteSize` or something like that?

(I know you're just following the fs naming, but this feels unclear to me and it's also inconsistent with `byteLength` where you clearly specify that the value is in bytes)

Reply:
actually i did outputSize because we use outputPath, and im not using a buffer to get the byte length so i went with size instead of byteLength. It was just a coincidence that it's the same as what fs.stat gives. :P but yeah sure can change to outputByteSize